gen3/neural-networks/object-detection/yolo-world/main.py

```diff
@@ -0,0 +1,255 @@
+import numpy as np
+import onnxruntime
+from transformers import AutoTokenizer
+import cv2
+import depthai as dai
+import argparse
+import requests
+import os
+
+MAJOR, MINOR = map(int, cv2.__version__.split(".")[:2])
+assert MAJOR == 4
```

Comment:
Would it be better to lock this in the requirements.txt so we don't need a check in the code?